.env.test

```diff
@@ -0,0 +1,21 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+API_KEY=TEST_KEY
+
+API_TOKEN=TEST_TOKEN
+
+TRELLO_TODO_LIST_ID=TEST_TRELLO_TODO_LIST_ID
+
+TRELLO_DONE_LIST_ID=TEST_TRELLO_DONE_LIST_ID
+
+TRELLO_DOING_LIST_ID=TEST_TRELLO_DOING_LIST_ID
+
+FLASK_SKIP_DOTENV=True
+
+TRELLO_BOARD_ID=TEST_TRELLO_BOARD_ID
```

Comment:
Minor, but your `.env.template` file lists this variable as `BOARD_ID` rather than `TRELLO_BOARD_ID` so it would be good to correct this there